pkgs/by-name/if/ifstate/package.nix

```diff
@@ -73,6 +107,16 @@ let
       jsonschema = "${self}/${python3Packages.python.sitePackages}/libifstate/schema/2/ifstate.conf.schema.json";
     };
 
+    outputs = [
+      "out"
+      "doc"
```

Comment:
How about using passthru instead of an output. That way patching ifstate does not require to build the docs.